suite/auto-sync/Update-Arch.sh

```diff
@@ -53,7 +53,7 @@ setup_build_dir() {
 # Main
 #
 
-supported="ARM"
+supported="ARM, PPC"
```

Comment:
After ARM64 it's probably better to convert this script to Python too

Reply:
Yes. Start with a script which updates the `inc` files today (it also applies the patches and updates the commit/tag note). Then, convert it before LLVM 17 rebase. This way we can test it directly.